src/config.ts

```diff
@@ -141,7 +141,9 @@ const proxy = {
 
 const store = {
 	country: envOrString(process.env.COUNTRY, 'usa'),
-	maxPrice: envOrNumber(process.env.MAX_PRICE),
+	maxPrice3070: envOrNumber(process.env.MAX_PRICE_3070),
+	maxPrice3080: envOrNumber(process.env.MAX_PRICE_3080),
+	maxPrice3090: envOrNumber(process.env.MAX_PRICE_3090),
```

Comment:
Users will need to know this is breaking change for `MAX_PRICE` though. Perhaps we should include a warning for now and then we can delete later.

```typescript
if (process.env.MAX_PRICE) logger.warn('ℹ MAX_PRICE is deprecated, please use MAX_PRICE_${series}')
```

Reply:
added relevant section to index with comments so other deprecated items can be added in the pre-execution